tests/test_dsl.py

```diff
@@ -0,0 +1,29 @@
+from cohortextractor.concepts import tables
+from cohortextractor.definition import Cohort, pick_first_value, register
+from cohortextractor.definition.base import registered_cohorts
+from cohortextractor.query_language import table
+from cohortextractor.query_utils import get_column_definitions
+
+
+def test_minimal_cohort_definition():
```

Comment:
This test is perfect. Right now it's a good check that we're really replicating the old DSL, but we can hopefully unit test all the DSL construction functionality like this by asserting against the semantic model that's generated.